xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -0,0 +1,86 @@
+require.config({
+    paths: {
+        moment: "$services.webjars.url('momentjs', 'moment.js')",
+        daterangepicker: "$services.webjars.url('bootstrap-daterangepicker', 'js/bootstrap-daterangepicker.js')"
+    },
+    shim: {
+        'daterangepicker': ['jquery', 'bootstrap', 'moment']
+    }
+});
+
+require(['jquery', 'moment', 'daterangepicker', 'xwiki-events-bridge'],
+function($, moment) {
+
+    // Load bootstrap-daterangepicker CSS only when needed
+    // This is a bit of a hack, but it's required until we have an API to
+    // include CSS from a webjar in Velocity.
+    $('head').append('<link rel="stylesheet" type="text/css" href="$services.webjars.url('bootstrap-daterangepicker', 'css/bootstrap-daterangepicker.css')">');
```

Comment:
A general remark: when you mix Velocity with JavaScript you need to make sure the result is still a valid JavaScript code. In the line above, as you can see from the syntax highlighting, the apostrophe opened in JavaScript is closed unintentionally in Velocity. Have you tried to build ``xwiki-platform-web``? My guess is that the build will fail because this file can't be minified.